src/main/resources/data.sql

```diff
@@ -14,6 +14,11 @@ VALUES (1, '2022-04-15 00:00:00', FALSE, 1, '2022-05-07 00:00:00', 'computer', T
        (2, '2022-03-17 00:00:00', FALSE, 2, '2022-05-20 00:00:00', 'sports', TRUE, 7, 'Shoe', 1, 'ACTIVE', 7, 2, 'Pieces'),
        (3, '2022-02-05 00:00:00', FALSE, 3, '2022-04-12 00:00:00', 'TV', TRUE, 43, 'TV', 1, 'ACTIVE', 43, 5, 'Pieces');
 
+INSERT INTO stock_details (i_date, quantity, price, remaining_quantity, product_id)
+VALUES ('2022-04-15 00:00:00', 2344, 999.99, 34525423, 3),
+       ('2022-01-15 00:00:00', 67, 99.99, 9687, 1),
```

Comment:
I wanna buy a few from your computers my friend, they are just 99.99 dollars ...:)

Reply:
Man, I dream of :)))